mqtt_bridge/app.py

```diff
@@ -34,10 +34,9 @@ def mqtt_bridge_node():
     # load bridge parameters
     bridge_dict_keys = ["factory","msg_type","topic_from","topic_to"]
     bridge_params = []
-    total_bridges = mqtt_node.get_parameter("n_bridges").value
-    for i in range(total_bridges):
-        bridge_n = str((i % total_bridges) + 1)
-        bridge_param = mqtt_node.get_parameter('bridge.bridge'+bridge_n).value
+    bridges_list = filter(lambda key: key[:7] == "bridge.", mqtt_node._parameters.keys())
```

Comment:
what is the significance of key[:7]?

Reply:
I printed out the mqtt_node._parameters dict, the key[:7] depends on the position of the parameters and this could create errors if people add more params on top of the file so handling with n_bridges param makes sense now as it would avoid possible errors in the future.